### DISCUSSION TITLE
Exporting images from Interfrom's visualisation

### DISCUSSION THREAD

**Discussion by MahdRaz** [alexisInSAR/EZ-InSAR#69] [Help]
First I'd like to thank you for this amazing GUI, then I was wondering if I can export images from the "visualization the interferograms" step after unwrapping. And by that I mean geolocation with value (lat,lon,value).
Thanks!

**Comment by alexisInSAR** [comment]
Hi, 

This option is not yet available. However, we are considering adding it.

Kind regards, 

Alexis
